app/assets/stylesheets/projects/index.scss

```diff
@@ -209,15 +220,48 @@ $demo-btn-shadow: #3C93D5;
   }
 }
 
-.projects-banner {
+#projects-image {
   background-image: image-url("projects/projects-banner.png");
   background-position: center;
   background-size: cover;
+  height: 365px;
+  width: 100%;
```

Comment:
Properties should be ordered background-image, background-position, background-size, height, position, top, width, z-index